src/chainparams.cpp

```diff
@@ -247,24 +247,26 @@ class CTestNetParams : public CChainParams {
         consensus.nMajorityRejectBlockOutdated = 75;
         consensus.nMajorityWindow = 100;
         consensus.BIP16Exception = uint256S("0x00000000dd30457c001f4095d208cc1296b0eed002427aa599874af7a432b105");
-        consensus.BIP34Height = 21111;
-        consensus.BIP34Hash = uint256S("0x0000000023b3a96d3484e5abb3755c413e7d41500f8e2a5c3f0dd01299cd8ef8");
+        consensus.BIP34Height = 14;
+        consensus.BIP34Hash = uint256S("0x079557d16edcd640c4057c9fddb81257263014fe384c4aa348c5b9d190650a46");
         // FIXME: adjust heights
-        consensus.BIP65Height = 1200000;
-        consensus.BIP66Height = 1200000;
+        consensus.BIP65Height = 120;
+        consensus.BIP66Height = 120;
         consensus.powLimit = uint256S("0000ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
         consensus.nPowTargetTimespan = 150;
+//        consensus.CSVHeight = 0;
+//        consensus.SegwitHeight = 120;
+//        consensus.MinBIP9WarningHeight = 2136; // segwit activation height + miner confirmation window
         consensus.nPowTargetSpacing = 150;
         consensus.nOriginalClaimExpirationTime = 262974;
         consensus.nExtendedClaimExpirationTime = 2102400;
-        consensus.nExtendedClaimExpirationForkHeight = 278160;
-        consensus.nAllowMinDiffMinHeight = 277299;
-        consensus.nAllowMinDiffMaxHeight = 1100000;
-        consensus.nNormalizedNameForkHeight = 993380;   // targeting, 21 Feb 2019
+        consensus.nExtendedClaimExpirationForkHeight = 1;
+        consensus.nAllowMinDiffMinHeight = 1;
+        consensus.nAllowMinDiffMaxHeight = 2;
+        consensus.nNormalizedNameForkHeight = 1;   // targeting, 21 Feb 2019
         consensus.nMinRemovalWorkaroundHeight = 99;
-        consensus.nMaxRemovalWorkaroundHeight = 100000000;
-        consensus.nWitnessForkHeight = 1198600;
```

Comment:
We don't want to delete this one!. It needs to be set, probably to block 120.

Reply:
wooops! didn't meant to get rid of it. fixed